scripts/ErrorReporter/errorreport.py

```diff
@@ -15,21 +15,39 @@
 else:
     raise RuntimeError("Unknown QT version: {}".format(qtpy.QT_VERSION))
 
-from qtpy import QtCore, QtGui, QtWidgets # noqa: E402
-from qtpy.QtCore import Signal # noqa: E402
-from qtpy.QtWidgets import QMessageBox # noqa: E402
-from mantidqt.utils.qt import load_ui # noqa: E402
+from qtpy import QtCore, QtGui, QtWidgets  # noqa: E402
+from qtpy.QtCore import Signal, QUrl  # noqa: E402
+from qtpy.QtGui import QDesktopServices  # noqa: E402
+from qtpy.QtWidgets import QMessageBox, QTextEdit  # noqa: E402
+from mantidqt.utils.qt import load_ui, import_qt  # noqa: E402
+
+MantidDesktopServices = import_qt('.._common', package='mantidqt.widgets', attr='MantidDesktopServices')
```

Comment:
Since this class may be useful to other parts of the code I'd be tempted to move this import to its own file. See the [`InterfaceManager`](https://github.com/mantidproject/mantid/blob/ef3ccb58e40735781011335e6b447a9ddd4d6e74/qt/python/mantidqt/interfacemanager.py) as an example. 

Also, as it is, I don't think this will work on a package build. The `.._common` argument needs to point to the `_common.sip` file from which the class is exposed. Personally I would put the new `mantiddesktopservices` file in the same directory as `interfacemanager.py` (note you'd only have *one* dot preceeding the `_common`).